### PR TITLE
VIH-7539 automate ejud hearing amend email

### PR DESCRIPTION
**Pull request by PrasannaDommalapati**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-7539

### Change description ###
- Added a new Notification Type 'HearingAmendmentEJudJoh'
- Added a new Migration for the notificationType

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by robertbennett1998 — DISMISSED** [review]
LGTM

File: NotificationApi/NotificationApi.IntegrationTests/Steps/HearingAmendmentSteps.cs [hunk 1]
```diff
@@ -41,6 +41,18 @@ public void GivenIHaveAHearingAmendmentForAnEjudJudgeEmailNotificationRequest()
             InitCreateNotificationRequest(request, _context);
         }
         
+        [Given(@"I have a hearing amendment for an ejud joh email notification request")]
+        public void GivenIHaveAHearingAmendmentForAnEjudJohEmailNotificationRequest()
+        {
+            var messageType = MessageType.Email;
+            var templateType = NotificationType.HearingAmendmentEJudJoh;
+            var parameters = InitGenericAmendmentParams();
+            parameters.Add("judge", $"{Faker.Name.FullName()}");
```
Comment (shaed-parkar) [inline NotificationApi/NotificationApi.IntegrationTests/Steps/HearingAmendmentSteps.cs:50]: You have the wrong param here. this is for the JOH, not a judge Copy and paste?

**Review by robertbennett1998 — APPROVED** [review]

File: NotificationApi/NotificationApi.IntegrationTests/Steps/HearingAmendmentSteps.cs (hunk 1 above)
Comment (PrasannaDommalapati) [reply NotificationApi/NotificationApi.IntegrationTests/Steps/HearingAmendmentSteps.cs:50]: > You have the wrong param here. this is for the JOH, not a judge Copy and paste?

doh, curse of copy-paste it is fixed now

**Review by robertbennett1998 — CHANGES_REQUESTED** [review]
As mentioned by Shaed

**Review by robertbennett1998 — APPROVED** [review]
Looks good

**Pull request merged**